docs/notebooks/dispersion/stix_dispersion.ipynb

```diff
@@ -0,0 +1,763 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "06f7f570",
+   "metadata": {},
+   "source": [
+    "# Stix Dispersion Solver\n",
+    "\n",
+    "[stix]: ../../api_static/plasmapy.dispersion.analytical.stix_.rst\n",
+    "[bellan2012]: https://doi.org/10.1029/2012JA017856\n",
+    "[stix1992]: https://link.springer.com/book/9780883188590\n",
+    "\n",
+    "This notebook details the functionality of the [stix()][stix] function. This is an analytical solution of equation 8 in [Bellan 2012][bellan2012], the function is defined by [Stix 1992][stix1996] in Section 1.2 to be:\n",
+    "\n",
+    "$$\n",
+    "        (S\\sin^{2}(\\theta) + P\\cos^{2}(\\theta))\\left ( \\frac{ck}{\\omega} \\right)^{4}\n",
+    "            - [\n",
+    "                RL\\sin^{2}(\\theta) + PS(1 + \\cos^{2}(\\theta))\n",
+    "            ]\\left ( \\frac{ck}{\\omega} \\right)^{2} + PRL = 0\n",
+    "$$\n",
+    "\n",
+    "where,\n",
+    "\n",
+    "$$\n",
+    "        \\mathbf{B_o} = B_{o} \\mathbf{\\hat{z}}   \n",
+    "         \\cos \\theta = \\frac{k_z}{k} \\\\\n",
+    "        \\mathbf{k} = k_{\\rm x} \\hat{x} + k_{\\rm z} \\hat{z}\n",
+    "$$\n",
+    "\n",
+    "$$\n",
+    "        S = 1 - \\sum_{s} \\frac{\\omega^{2}_{p,s}}{\\omega^{2} -\n",
+    "            \\omega^{2}_{c,s}}\\hspace{2.5cm}\n",
+    "        P = 1 - \\sum_{s} \\frac{\\omega^{2}_{p,s}}{\\omega^{2}}\\hspace{2.5cm}\n",
+    "        D = \\sum_{s}\n",
+    "            \\frac{\\omega_{c,s}}{\\omega}\n",
+    "            \\frac{\\omega^{2}_{p,s}}{\\omega^{2} -\n",
+    "            \\omega_{c,s}^{2}}\n",
+    "$$\n",
+    "\n",
+    "$$\n",
+    "        R = S + D \\hspace{1cm} L = S - D\n",
+    "$$\n",
+    "\n",
+    "$\\omega$ is the wave frequency, $k$ is the wavenumber, $\\theta$ is the wave propagation angle with respect to the background magntic field $\\mathbf{B_{0}}$, $s$ corresponds to plasma species $s$, $\\omega_{p,s}$ is the plasma frequency of species and $\\omega_{c,s}$ is the gyrofrequency of species $s$.\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "Note\n",
+    "\n",
+    "The derivation of this dispersion relation assumed:\n",
+    "    \n",
+    " * zero temperature for all plasma species ($T_{s}=0$)\n",
+    " * quasi-neutrallity\n",
```

Comment:
```suggestion
    " * quasineutrality\n",
```
Come to think of this, this might be good to add to our glossary too (but not this PR)